packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -134,6 +134,10 @@ class SliverMultiBoxAdaptorParentData extends SliverLogicalParentData with Conta
   /// [RenderSliverMultiBoxAdaptor._keepAliveBucket].
   bool _keptAlive = false;
 
+  /// Whether the widget is currently being kept alive, i.e. has [keepAlive] set
+  /// to true and is offscreen.
+  bool get keptAlive => _keptAlive;
```

Comment:
normally we put the public getter above the private field and only have docs for the public getter. your call as to whether this case warrants an exception.